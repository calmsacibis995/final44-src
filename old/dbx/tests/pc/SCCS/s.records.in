h15798
s 00005/00000/00000
d D 5.1 90/06/25 15:37:27 bostic 1 0
c date and time created 90/06/25 15:37:27 by bostic
e
u
U
t
T
I 1
whatis Rec
whatis r
run
print r
print r.realValue, r.subrange, r.intValue, r.charValue
E 1
