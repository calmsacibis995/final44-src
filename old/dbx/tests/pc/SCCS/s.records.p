h27865
s 00016/00000/00000
d D 5.1 90/06/25 15:37:29 bostic 1 0
c date and time created 90/06/25 15:37:29 by bostic
e
u
U
t
T
I 1
program recordtest (input, output);
type
    Rec = record
	charValue : char;
	intValue : integer;
	subrange : 0..1000;
	realValue : real;
    end;
var
    r : Rec;
begin
    r.charValue := 'c';
    r.intValue := 3;
    r.subrange := 10;
    r.realValue := 3.4;
end.
E 1
