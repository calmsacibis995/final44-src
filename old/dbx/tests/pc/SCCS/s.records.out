h28048
s 00013/00000/00000
d D 5.1 90/06/25 15:37:28 bostic 1 0
c date and time created 90/06/25 15:37:28 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
type Rec = record
    charValue : char;
    intValue : integer;
    subrange : 0..1000;
    realValue : real;
end;
var r : Rec;

execution completed
(charValue = 'c', intValue = 3, subrange = 10, realValue = 3.4) 
3.4 10 3 'c' 

E 1
